S2VX.Game.Tests/HeadlessTests/ScoreProcessorTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Screens;
+using osu.Framework.Testing;
+using S2VX.Game.Play;
+using S2VX.Game.Play.UserInterface;
+using S2VX.Game.Story;
+using S2VX.Game.Story.Note;
+using System.IO;
+
+namespace S2VX.Game.Tests.VisualTests {
```

Comment:
Is it supposed to be S2VX.Game.Tests.VisualTests here? I can't find these tests under the testing UI or test explorer.

Reply:
Addressed in new patch